Sources/MapboxNavigation/NavigationMapView+VanishingRouteLine.swift

```diff
@@ -122,6 +124,24 @@ extension NavigationMapView {
         return RouteLineGranularDistances(distance: distance, distanceArray: indexArray.compactMap{ $0 })
     }
     
+    func findDistanceToNearestPointOnCurrentLine(coordinate: CLLocationCoordinate2D, granularDistances: RouteLineGranularDistances, upcomingIndex: Int) -> CLLocationDistance {
+        guard granularDistances.distanceArray.indices.contains(upcomingIndex) else { return 0.0 }
+
+        var coordinates = [CLLocationCoordinate2D]()
+        for index in max(0, upcomingIndex - 10)...upcomingIndex  {
```

Comment:
Why `10` specifically in this case? Maybe also makes sense to provide a comment, which describes your motivation.

Reply:
We could also take other numbers for the passed points, but to keep the parity with Android. It's to add extend to sliced route line a little longer incase the curved shape of the route.